static/app/views/performance/transactionSummary/transactionSpans/content.tsx

```diff
@@ -56,7 +53,7 @@ type Props = {
 };
 
 function SpansContent(props: Props) {
-  const {location, organization, eventView, projectId, setError, transactionName} = props;
+  const {location, organization, eventView, projectId, transactionName} = props;
```

Comment:
Will we no longer call `setError` if the query errors (looks like we're just treating it as [] suspectSpans)? 
In this case, looks like `setError` from the props is unused, should we just remove it?

Reply:
Leaving this, will be coming back to show the errors.